website/docs/r/service_principal.html.markdown

```diff
@@ -36,6 +38,8 @@ The following arguments are supported:
 
 * `application_id` - (Required) The ID of the Azure AD Application for which to create a Service Principal.
 
+* `tags` - (Optional) A list of tags to apply to the Service Principal.
```

Comment:
since this is ForceNew we'll want to suffix this with `Changing this forces a new resource to be created`?